lib/src/participant/remote_participant.dart

```diff
@@ -175,7 +175,11 @@ class RemoteParticipant extends Participant {
   }
 
   @override
-  Future<void> unpublishTrack(String trackSid, {bool notify = true}) async {
+  Future<void> unpublishTrack(
+    String trackSid, {
+    bool notify = true,
+    bool? stopTrack,
```

Comment:
as DL mentioned, I don't think this makes sense for remote. ideally this is exposed only to the package and not outside of the library.